.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 executors:
   linux-build:
       docker:
-        - image: cimg/base:2022.09
+        - image: cimg/base:2022.08
```

Comment:
I'm going to push a one-line change to make this `cimg/base:2022.08-20.04`. That's the same image, but at makes it clearer that we're using Ubuntu 20.04 as the build target.